code/modules/mob/living/carbon/human/human.dm

```diff
@@ -237,7 +237,7 @@
 
 
 /mob/living/carbon/human/attack_animal(mob/living/M as mob)
-	if(M.melee_damage_upper == 0)
+	if(M.melee_damage_upper += burn_damage_upper == 0)
```

Comment:
???

Reply:
that code is checking the dmg source, to see if damage is being dealt. I added a second dmg source, so i needed it to be checked too.